### PR TITLE
test: Add Tests for Protocol Fee Getters and Setters

### LINKED ISSUE #25: Add Tests for Protocol Fee Getters and Setters
Add test coverage for the protocol fee management functions in the Distributor contract.

Functions to test:
- `get_protocol_fee_percent()`
- `set_protocol_fee_percent()`
- `get_protocol_fee_address()`
- `set_protocol_fee_address()`

Test cases should include:
- Setting and getting fee percentage
- Setting and getting fee address
- Only the owner can set values
- Invalid inputs (if any)
- Edge cases (0%, 100%)

Related to PR #24 which implemented protocol fees.

### PR DESCRIPTION
**Pull request by No-bodyq**
closes #25

### CODE REVIEW

File: tests/test_distribute_contract.cairo [hunk 1]
```diff
@@ -7,11 +7,16 @@ use snforge_std::{
 use fundable::interfaces::IDistributor::{IDistributorDispatcher, IDistributorDispatcherTrait};
 use openzeppelin::token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTrait};
 
+pub fn OWNER_ADDRESS() -> ContractAddress {
```
Comment (Utilitycoder) [inline tests/test_distribute_contract.cairo:10]: Use the standard ```contract_address_const``` for a new address. It aligns with our style

File: tests/test_distribute_contract.cairo [hunk 2]
```diff
@@ -7,11 +7,16 @@ use snforge_std::{
...
+    'owner'.try_into().unwrap()
+}
+
 fn setup() -> (ContractAddress, ContractAddress, IDistributorDispatcher) {
     let sender: ContractAddress = contract_address_const::<'sender'>();
     // Deploy mock ERC20
     let erc20_class = declare("MockUsdc").unwrap().contract_class();
-    let mut calldata = array![sender.into(), sender.into()];
+    let owner = OWNER_ADDRESS();
```
Comment (Utilitycoder) [inline tests/test_distribute_contract.cairo:18]: Change this to ```contract_address_const```

Comment (No-bodyq) [reply tests/test_distribute_contract.cairo]: Done @Utilitycoder

**Review by Utilitycoder — COMMENTED** [review]
Thanks for the awesome job
